### PR TITLE
EC2: Introduction

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
+    parser.addoption("--type", action="store", default="aks",
+                     help="PCP selection: aks/eks")
+    parser.addoption("--nocleanup", action="store_true",
+                     default=False, help="Disable cleanup")
+    parser.addoption("--skipterraform", action="store_true",
+                     default=False, help="Not invoke terraform at all")
+    parser.addoption("--imageid", action="store",
+                     default="", help="Image id name")
+    parser.addoption("--bloburi", action="store",
+                     default="", help="URL to image blob")
```
Comment (jlausuch) [inline conftest.py:18]: Not very important, but wouldn´t it be more readable using underscores? `image_id` `no_cleanup` `skip_terraform` `blob_uri`

File: terraform/ec2.tf [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+provider "aws" {
+    region = var.region
+}
+
+variable "instance_count" {
+    default = "1"
+}
+
+variable "name" {
+    default = "openqa-aks"
```
Comment (jlausuch) [inline terraform/ec2.tf:14]: eks?

File: conftest.py (hunk 1 above)
Comment (asmorodskyi) [reply conftest.py:18]: ok

File: terraform/ec2.tf (hunk 2 above)
Comment (asmorodskyi) [reply terraform/ec2.tf:14]: good catch :)

**Comment by jlausuch** [comment]
LGTM

**Pull request merged**